lib/middleware.js

```diff
@@ -15,6 +15,9 @@ module.exports = function(self, options) {
     middleware: function(req, res, next) {
       var to, host, matches, locale, prefix, hostname, subdomain, domain, candidates;
 
+      req.data.workflow = req.data.workflow || {};
```

Comment:
Making as much data about workflow as possible available in other contexts, although we can't have access to the "context" property that localizations really want outside of a full page request.